widgets/open-tickets/data.js

```diff
@@ -1,50 +1,50 @@
-/**
- * Load Page
- */
-function loadPage()
-{
-    jQuery.ajax("./src/index.php?widgetId=" + widgetId + "&page=" + pageNumber, {
-        success: function (htmlData) {
-            jQuery("#openTicketsTable").empty().append(htmlData);
-            var hostMonitoringTable = jQuery("#openTicketsTable").find("img, style, script, link").load(function () {
-                var h = document.getElementById("openTicketsTable").scrollHeight + 50;
-                parent.iResize(window.name, h);
-            });
-        }
-    });
-    if (autoRefresh) {
-        if (timeout) {
-            clearTimeout(timeout);
-        }
-        timeout = setTimeout(loadPage, (autoRefresh * 1000));
-    }
-}
-
-/**
- * Load toolbar
- */
-function loadToolBar()
-{
-    jQuery("#toolBar").load(
-            "./src/toolbar.php",
-            {
-                widgetId: widgetId
-            }
-    );
-}
-
-
-jQuery(function () {
-    loadToolBar();
-    loadPage();
-    $('.checkall').live('click', function () {
-        var chck = this.checked;
-        $(this).parents().find(':checkbox').each(function () {
-            $(this).attr('checked', chck);
-            clickedCb[$(this).attr('id')] = chck;
-        });
-    });
-    $(".selection").live('click', function () {
-        clickedCb[$(this).attr('id')] = this.checked;
-    });
-});
+/**
+ * Load Page
+ */
+function loadPage()
+{
+    jQuery.ajax("./src/index.php?widgetId=" + widgetId + "&page=" + pageNumber, {
```

Comment:
JS coding style specify a 2 spaces indentation

Reply:
> JS coding style specify a 2 spaces indentation

Humm .. so we need to change this document?
https://github.com/centreon/centreon/blob/master/doc/coding-style/js/README.md